spec/controllers/api/v1/trucking/driver_commissions_history_controller_spec.rb

```diff
@@ -0,0 +1,148 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::Trucking::DriverCommissionsHistoryController,
+               :type => :controller do
+  let(:driver_commission_history) do
+    [
+      {
+        :driver_id => Faker::Number.number(4),
+        :backhauls => 'BACKHAULS',
+        :freight_revenue => Faker::Number.number(4),
+        :customer_id => 0001,
+        :delivery_date => Date.new(2012, 01, 01),
+        :driver_rate => Faker::Number.number(4),
+        :freight_bill_number => Faker::Number.number(4),
+        :fuel_revenue => Faker::Number.number(4),
+        :fuel_surcharge => Faker::Number.number(4),
+        :layover => 'layover',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.